packages/react-tinacms-editor/src/components/Wysiwyg/index.tsx

```diff
@@ -85,3 +98,36 @@ const Wrapper = styled.div`
   display: block;
   min-height: 100px;
 `
+
+function generateImageProps(
```

Comment:
What if this was `useImageProps` and the `useMemo` was called from in here?

Reply:
I like that!